src/routes/health.rs

```diff
@@ -0,0 +1,10 @@
+use crate::utils::context::Context;
+use crate::config::{about_cache_duration, base_transaction_service_url};
+use crate::utils::cache::CacheExt;
+use crate::utils::errors::ApiResult;
+
+#[get("/health")]
+pub fn health(context: Context) -> ApiResult<String> {
+    let url = format!("{}/about", base_transaction_service_url());
+    context.cache().request_cached(context.client(), &url, about_cache_duration())
```

Comment:
I would not do this, as this would make the health of this service depend on the health of the transaction service ... which is wrong in the context of deployment (where we want to know if the service is ready to being used). To be fair the same argument can be made about the redis but I would put them on different levels ... maybe :P

I would really just return a empty response. (with or without caching)

cc @Uxio0 what is your take on this?

Reply:
Totally agree @rmeissner. Basically this `health` endpoint is important so the services can be monitored. For example, if the service is not responding for some configured time it will get rebooted or another replica will be set up. It doesn't make sense that it depends on the transaction service `health`.

We use `about` endpoint on the transaction service as it's cached, easy to return and also it if it's not working is because there's a configuration problem.

> I would really just return a empty response. (with or without caching)
That should work, as long as it's true that if this endpoint is working the service is working